docs/Troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting
+
+This page aimed to help you fix the common problems encountered while using Spaceship prompt.
+
+## What's the weird symbol for `git` branch?
+
+You need to have a powerline patched font inorder to properly display `git` branch symbol.
+
+* Install any powerline compatible font like [Fira Code](https://github.com/tonsky/FiraCode) or [others](https://github.com/powerline/fonts).
+* Configure your terminal emulator to [use that font](https://powerline.readthedocs.io/en/master/troubleshooting/osx.html).
+
+## What's the weird characters infront of sections?
+
+This is not an issue with Spaceship prompt. Spaceship uses Unicode symbols to represent `SPACESHIP_*_SYMBOL` in sections. To solve this problem:
+
+* Verify your terminal emulator support Unicode characters with this command:
+  ```zsh
+  curl http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt
+  # or
+  wget -O - http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt
+  ```
+* Configure your terminal emulator to use UTF-8 as character encoding.
+
+## Some of sections icons put on one another?
```

Comment:
Some section icons overlap each other?

Reply:
Fixed. Perks of being a non-native speaker 😁